### PR TITLE
[aravis] new port

### PR DESCRIPTION
**Pull request by cenit**
Depends on: #13100

### CODE REVIEW

File: ports/aravis/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                FILENAME "tools/python/${PYTHON3_DIR_NAME}/get-pip.py"
+                SHA512 92e68525830bb23955a31cb19ebc3021ef16b6337eab83d5db2961b791283d2867207545faf83635f6027f2f7b7f8fee2c85f2cfd8e8267df25406474571c741
+            )
+            execute_process(COMMAND ${PYTHON3_DIR}/python${EXECUTABLE_SUFFIX} ${GET_PIP})
+        endif()
+        execute_process(COMMAND ${PYTHON3_DIR}/Scripts/pip${EXECUTABLE_SUFFIX} install python-gettext --user)
+    else()
+        execute_process(COMMAND ${PYTHON3_DIR}/easy_install${EXECUTABLE_SUFFIX} python-gettext)
+    endif()
+endif()
```
Comment (vicroms) [inline ports/aravis/portfile.cmake]: It almost feels like this should be made into a vcpkg helper function.

Comment (cenit) [reply ports/aravis/portfile.cmake]: Yes. Also, this PR was opened too soon. Maybe it’s better if I just close it. I don’t have much time to work on it now, and the required work is a lot before being ready.

**Comment by LilyWangL** [comment]
Pinging @cenit for response. Is work still being done for this PR?

**Comment by LilyWangL** [comment]
Closing this PR since it seems that no progress is being made. Please reopen if work is still being done.

**Pull request closed without merging**